TWLight/resources/helpers.py

```diff
@@ -1,9 +1,28 @@
 from django.conf import settings
 from numbers import Number
 import json
+import logging
 import os
 
 
+"""
```

Comment:
Let's put this into a function. This helper file has several functions that are unrelated to translation code. Whenever we import this file, this segment will run regardless of whether it is needed or not. If we move this to a function, we can call it as needed.